test/e2e/page-objects/pages/home/homepage.ts

```diff
@@ -146,8 +149,10 @@ class HomePage {
     await this.driver.clickElement(this.sendButton);
   }
 
-  async startBridgeFlow(): Promise<void> {
-    await this.driver.clickElement(this.bridgeButton);
+  async startBridgeFlow(isSolana: boolean = false): Promise<void> {
+    isSolana
+      ? await this.driver.clickElement(this.nonEvmBridgeButton)
+      : await this.driver.clickElement(this.bridgeButton);
```

Comment:
mmm I  think this is a bit confusing, as I see we already have a class for non-evm homepage, so if we wanted to start a non evm bridge we should use the non evm hompage method (if there's none, then we should add a method there with its corresponding selector) and leave this method as it was.
Wdyt?

Reply:
you are right, this was done at the very beginning in order to use the existing flow, but since it was not using POM and button ids change between evm and non-evm, I implemented it by my own. Fixed